deps/v8/src/builtins/generate-bytecodes-builtins-list.cc

```diff
@@ -41,6 +41,14 @@ void WriteBytecode(std::ofstream& out, Bytecode bytecode,
 void WriteHeader(const char* header_filename) {
   std::ofstream out(header_filename);
 
+#ifdef CCACHE_USED
+  // Write a cache invalidator to ensure that ccache does not cache this file.
```

Comment:
I wonder why is it not a problem for non-Windows. Does it have to do with the fact that we only use PCH on Windows? What happens if you just put ccache_used into the condition here?

https://github.com/nodejs/node/blob/45732314d471836b238c78e76a9d3aee7bda9381/tools/v8_gypfiles/v8.gyp#L51

Reply:
Yes, this is connected to PCH usage. The idea of disabling PCH when wanting to use ccache did cross my mind, but from what I recall, the time for compilation without PCH was much greater than with it. Not sure about the numbers. Let me reiterate it and then I'll share the exact numbers from my machine here together with the PCH error I get from cache. Thanks for the suggestion.

Reply:
My understanding is that PCH would help a deal when there's no good caching but wouldn't matter as much when there is caching? Especially when dealing with V8 which uses templates very extensively, and headers change quite often...